Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM trailofbits/polytracker-llvm:f1385f079c2573f2b31834354310e7265a9bb4b3
+FROM trailofbits/polytracker-llvm:16daa680dcff0dc86ebf6ae81f5382859695cb35
```

Comment:
Can you remove `sqlite3` and `libsqlite3-dev` from the installed packages, or are they still needed?

Reply:
Yes! Good point! That should work. Also, libjpeg-dev shouldn't be needed. Maybe others.